server/paymentProviders/opencollective/host.js

```diff
@@ -16,6 +18,24 @@ paymentMethodProvider.getBalance = () => {
   return Promise.resolve(maxInteger);
 };
 
+paymentMethodProvider.createPlatformTipTransaction = async payload => {
+  const transaction = payload.transaction;
+  const TransactionGroup = transaction.TransactionGroup || uuid();
```

Comment:
:warning: We should not have this `|| uuid()`. If there's no `TransactionGroup` we can't create the platform tip, as platform tips need to be attached to an existing transaction. This is especially true since the `TransactionGroup` is not passed in the payload from https://github.com/opencollective/opencollective-api/pull/5402/files#diff-1ffaf5a7e2c0ad8ce588e2ebdcb1ddf56a3c2fe93e7f52043c2b0082412fe6b7R76, so the platform tip won't be linked to the original transaction.

Reply:
Done. 👍🏽